src/compiler/VendorCompiler.ts

```diff
@@ -219,6 +222,95 @@ export class VendorCompiler implements ICompiler {
                             'file-loader'
                         ]
                     },
+                    {
+                        // jquery.ba-bbq expects 'this' to be th 'window' object
+                        // see https://v4.webpack.js.org/guides/shimming/#granular-shimming
+                        test: /jquery.ba-bbq\.js/,
+                        use: 'imports-loader?wrapper=window',
+                    },
+                    // the following modules are used as globals througout cplace, so they are exposed
+                    // see https://v4.webpack.js.org/loaders/expose-loader/
+                    {
+                        test: /underscore/,
+                        loader: 'expose-loader',
```

Comment:
I think this whole thing should not be hardcoded in the `cplace-asc`. I think it should be more agnostic and not know about the `cplace` dependencies. It should probably get/extend them using some configuration. Solution for this situation could be having the `webpack.config.js` in the plugin assets folder. cplace-asc would then just extend this configuration. Just small reason why...If we remove `underscore` as a dependency, this code would be obsolete.

Reply:
By "thing" I mean all the dependencies listed here